support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -174,32 +188,84 @@ function WeeklyCheckoutForm(props: PropTypes) {
 		props.setBillingCountry(props.deliveryCountry);
 	};
 
+	const inThreeTierTestVariant =
+		props.participations.threeTierCheckout === 'variant';
+
 	const paymentMethods = supportedPaymentMethods(
 		props.currencyId,
 		props.billingCountry,
 	);
 
+	const billingPeriodNounLowerCase = billingPeriodNoun(
+		props.billingPeriod,
+	).toLowerCase() as 'month' | 'year';
+
+	const digitalPlusPrintBillingPeriod = props.billingPeriod.toLowerCase() as
+		| 'monthly'
+		| 'annual';
+
+	const standardDigitalPlusPrintPrice =
+		tierCards.tier3.plans[digitalPlusPrintBillingPeriod].charges[
+			props.countryGroupId
+		].price;
+	const digitalPlusPrintPotentialDiscount =
+		tierCards.tier3.plans[digitalPlusPrintBillingPeriod].charges[
+			props.countryGroupId
+		].discount;
+
+	const publicationStartDays = days.filter((day) => {
+		const invalidPublicationDates = ['-12-24', '-12-25', '-12-30'];
```

Comment:
12-30 seems odd publication invalid date - must be right?

Reply:
I stole these from the existing functionality, I assume they are correct